README.md

```diff
@@ -12,90 +12,110 @@
   <img alt="GitHub closed issues" src="https://img.shields.io/github/issues-closed/posthog/posthog"/>
 </p>
 
-## PostHog is an open-source product analytics suite, built for engineers
-* Automatically track every event on your website or app
-* Understand your users and how to improve your product
-* Deploy on your own infrastructure to keep control of your data.
+<p align="center">
+  <a href="https://posthog.com/docs">Docs</a> - <a href="https://posthog.com/using-posthog">Using PostHog</a> - <a href="https://posthog.com/slack">Support community</a> - <a href="https://posthog.com/roadmap">Roadmap</a> - <a href="https://github.com/PostHog/posthog/issues/new?assignees=&labels=bug&template=bug_report.md">Bug report</a>
+</p>
+
+## PostHog is an open-source suite of product and data tools, built for engineers
+
+- Automatic and/or customized event capture for your app and website
+- Analyze your data with visualizations and session recordings
+- Improve your product with A/B testing and feature flags
+- Keep control over your data by deploying PostHog on your infrastructure
+- Use apps to connect to external services and manage data flows
 
+## Table of Contents
 
+- [Get started for free](#get-started-for-free)
+- [Features](#features)
+- [Docs and support](#docs-and-support)
+- [Contributing](#contributing)
+- [Philosophy](#philosophy)
+- [Open-source vs paid](#open-source-vs-paid)
```

Comment:
May be a stretch goal, but I think could be worth adding a section that compares PostHog to Amplitude, Heap, Logrocket, etc. Could just be a summary, then link to blog content.